packages/bsky/tests/views/threadgating.test.ts

```diff
@@ -29,6 +29,19 @@ describe('views with thread gating', () => {
     await network.close()
   })
 
+  // check that canReply state is applied correctly in a simple method like getPosts
+  const checkCanReplyForUser = async (
```

Comment:
I felt that this was an economical option to ensure that `canReply` state is being applied to all views correctly

Other option is to either augment the seed client to include some threadgates or write a bunch of thorough tests on every route that hydrates posts.

`getPosts` is the simplest hydration method for posts & the same hydration logic is used for every route, so the thinking is that if it is applied correctly here, that it should be applied correctly elsewhere as well